### PR TITLE
change to new cython version

### PR DESCRIPTION
**Pull request by wkerzendorf**
@unoebauer, @scoder
version 0.28.4 should have the right fix in there. 

cython/cython#2199

### CODE REVIEW

**Comment by wkerzendorf** [comment]
@unoebauer - should we merge?

**Comment by unoebauer** [comment]
I think we should - Travis is happy with both Linux and OSX builds, so I don't see a reason why we shouldn't. We might wait for @Heringer-Epson though to check whether this version works on his local installation.

**Comment by wkerzendorf** [comment]
@unoebauer - no I removed the OSX build for now.

**Comment by wkerzendorf** [comment]
I would merge and put a reminder in my phone to check in 3 months to reinclude the OSX build

**Comment by unoebauer** [comment]
Sorry, I missed that the OSX is deactivated. So on OSX the segfault is still triggered? And locally on your OSX build it works?

**Comment by wkerzendorf** [comment]
@unoebauer it doesn't segfault in the debugger even. I think, I will just disable it for now.

**Comment by Heringer-Epson** [comment]
Sorry for the delay. I will check this today.

**Comment by wkerzendorf** [comment]
@Heringer-Epson be sure to delete montecarlo.c (look at it - it will say `generated with Cython VERSION_NUMBER` then rerun the compilation.

File: .travis.yml [hunk 1]
```diff
@@ -87,15 +88,9 @@ install:
    - cd $TRAVIS_BUILD_DIR
    - conda env create -f tardis_env27.yml
    - source activate tardis
-   #trouble with building due to segfault at cython (https://github.com/cython/cython/issues/2199)
-   #remove if we can get normal cython through conda
```
Comment (unoebauer) [inline .travis.yml]: Should we leave this in as a comment? This way, we'll know what to do when some strange cython error appears in the future.

Comment (wkerzendorf) [reply .travis.yml]: That’s why I put it in

Comment (unoebauer) [reply .travis.yml]: I was thinking of keeping the procedure of getting bleeding-edge cython via github and compiling it with custom flags as a comment block and providing some additional information.

**Comment by Heringer-Epson** [comment]
Yes, this worked upon trying to rerun the installation. Note, using the conda env suggestion in tardis_env27.yml triggers an error due to Pyqt not being installed (I believe it's required for the gui interface). Installing Pyqt=4.11 on the local env then caused a bunch of packages to be downgraded.

**Comment by wkerzendorf** [comment]
that's interesting. anyways. I'll merge once I add the manual cython installing instructions in.

**Pull request merged**